force-app/main/default/flows/After_Insert_Event.flow-meta.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <actionCalls>
+        <name>Send_Email_Reminder</name>
```

Comment:
@bmtemakina the name of the flow is not convenient since it's triggered after insert and update